front/app/containers/EventPage/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import messages from './messages';
+
+// components
+import ContentContainer from 'components/ContentContainer';
+import SectionContainer from 'components/SectionContainer';
+import { Helmet } from 'react-helmet';
+import EventViewer from './EventViewer';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// styling
+import styled from 'styled-components';
+
+const StyledContentContainer = styled(ContentContainer)`
+  max-width: calc(${(props) => props.theme.maxPageWidth}px - 100px);
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const EventViewerWithTopMargin = styled(EventViewer)`
```

Comment:
For code maintainability, I would not add the styles in the component name. If you need to change the styles, you also need to change the name. I'd go with `StyledEventViewer`.